cpu/stm32f1/include/cpu_conf.h

```diff
@@ -36,12 +37,122 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE      (2048U)
+
+#if defined(CPU_MODEL_STM32F103CB) || defined(CPU_MODEL_STM32F103RB)
+#define FLASHPAGE_NUMOF     (64U)
+#elif defined(CPU_MODEL_STM32F103RE)
+#define FLASHPAGE_NUMOF     (256U)
+#endif
```

Comment:
tip: we solved this in one project by defining them in the linker file and the define was just pointing there.